Documentation/kubelet-wrapper.md

```diff
@@ -61,3 +95,4 @@ ExecStart=/opt/bin/kubelet-wrapper \
 
 [#2141]: https://github.com/coreos/rkt/issues/2141
 [kubelet-wrapper]: https://github.com/coreos/coreos-overlay/blob/master/app-admin/kubelet-wrapper/files/kubelet-wrapper
+[addon-logging]: https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/fluentd-elasticsearch
```

Comment:
Might be preferable to target the release branch: https://github.com/kubernetes/kubernetes/tree/release-1.2/cluster/addons/fluentd-elasticsearch

This will also be easily picked up in a grep for `1.2`